src/algorand_hash.rs

```diff
@@ -10,6 +10,7 @@ use crate::{
     algorand_errors::AlgorandError,
     algorand_genesis_id::AlgorandGenesisId,
     algorand_types::{Byte, Bytes, Result},
+    crypto_utils::base32_encode_with_no_padding,
```

Comment:
Clippy is complaining about this. Maybe we should remove `#[cfg(test)]` from the two functions you added.

Reply:
No need - just remove the import from there and add it on it's own line:

```rust
#[cfg(test)]
use crate::crypto_utils::base32_encode_with_no_padding;
```